USERS.md

```diff
@@ -282,3 +282,4 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Yubo](https://www.yubo.live/)
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [GoTo Financial](https://gotofinancial.com/)
```

Comment:
I'm unable to get the site to resolve. Can you double-check the URL?

Could you also move the item up into alphabetical order?

Reply:
The site resolves for me at least.